tests/js/common/shell/shell-pregel-components.js

```diff
@@ -34,8 +34,21 @@ var db = require("@arangodb").db;
 var graph_module = require("@arangodb/general-graph");
 var internal = require("internal");
 var console = require("console");
-var EPS = 0.0001;
 let pregel = require("@arangodb/pregel");
+let pregelTestHelpers = require("@arangodb/graph/pregel-test-helpers");
+
+// TODO make this more flexible: input maxWaitTimeSecs and sleepIntervalSecs
+const pregelRunSmallInstance = function (algName, graphName, parameters) {
+  const pid = pregel.start("wcc", graphName, parameters);
+  const maxWaitTimeSecs = 120;
+  const sleepIntervalSecs = 0.2;
+  let wakeupsLeft = maxWaitTimeSecs / sleepIntervalSecs;
+  while (pregel.status(pid).state !== "done" && wakeupsLeft > 0) {
+    wakeupsLeft--;
+    internal.sleep(0.2);
+  }
+  return pregel.status(pid);
+};
```

Comment:
Import this function from `pregel-test-helpers.js`

Reply:
Done